repos/system_upgrade/el7toel8/actors/checkleftoverpackages/tests/test_checkleftover_checkleftoverpackages.py

```diff
@@ -1,7 +1,14 @@
+import distro
+import pytest
+
 from leapp.models import LeftoverPackages
-from leapp.snactor.fixture import current_actor_context
 
 
+@pytest.mark.skipif(
+    distro.linux_distribution()[0]
+    not in ('Fedora', 'CentOS Linux', 'Red Hat Enterprise Linux Server'),
```

Comment:
the PR is not doing what is telling.  there are more rpm based systems than in the list (e.g. [here](https://en.wikipedia.org/wiki/Category:RPM-based_Linux_distributions)). instead of this, rather check existence of  the `/usr/bin/rpm` file, if you want to be more precise. Anyway, this is still wrong as the test is just skipped instead of to be allowed to run. Basically this is incorrect test at all, as it is affected by environment and it not testing anything except the point it does not crash during the execution.

Reply:
then I suggest removing it @pirat89 . Agree with your statements